apps/mosaico/src/Main.purs

```diff
@@ -307,9 +311,27 @@ frontpage env { guards: { credentials } } = do
             appendMosaico mosaicoString env.htmlTemplate >>= appendHead (mkWindowVariables windowVars)
   pure $ maybeInvalidateAuth user $ htmlContent $ Response.ok $ StringBody html
 
-mkArticleFeed :: Maybe String -> String -> Array ArticleStub -> String
-mkArticleFeed feedPage feedType feedContent =
-  stringify $ encodeJson { feedPage, feedType, feedContent: encodeStringifyArticleStubs feedContent }
+getFrontPage :: Paper -> String -> Aff ArticleFeed
```

Comment:
We've used frontpage as "Frontpage" in other camel cased contexts, like in "FrontpageFeed". Let's stick to the same convention with the new names that this PR introduces, so this function would be `getFrontpage`. The same goes for other FrontPages.